lib/cdo/hamburger.rb

```diff
@@ -320,10 +318,11 @@ def self.get_header_contents(options)
         header_links.concat(about_intl)
       end
     else
+      header_links = signed_out_links
       if options[:language] == "en"
-        header_links = en_signed_out_links
+        header_links.concat(about)
       else
-        header_links.concat(non_en_signed_out_links).concat(about_intl)
+        header_links.concat(about_intl)
```

Comment:
We no longer need a different international about. Everyone should get the normal about